api/src/main/java/javax/enterprise/inject/spi/InterceptionFactory.java

```diff
@@ -88,7 +88,7 @@
      *
```

Comment:
I would rather modify the first sentense to something like:

Instructs the container to ignore all non-static, final methods with public, protected or default visibility declared by any class in the type hierarchy of the intercepted instance during invocation of `createInterceptedInstance()`.